src/ros/laser_scan_observer.h

```diff
@@ -8,13 +9,22 @@
 #include "../core/sensor_data.h"
 #include "topic_with_transform.h"
 
+/**
+ * \brief This class observes for LaserScan and stores information about location.
```

Comment:
Better: "Observes laser scans and odometry, converts ROS structures to internal representation."